baselines/jft/active_learning.py

```diff
@@ -832,6 +834,9 @@ def write_note(note):
 
   def _main(argv):
     del argv
+
+    global JAX_CPU
+    JAX_CPU = jax.devices("cpu")[0]
```

Comment:
Instead of a global here, can you either directly use `jax.devices("cpu")[0]` up above in get_ids_logits_mask or define a helper function (e.g., _get_cpu_device) and use that?